views/posts/index.hbs

```diff
@@ -12,7 +12,13 @@
     <li class="post">
       {{this.timestamp}}
```

Comment:
an Idea - would be what timestamp displayed underneath user profile? i.e.
- Username
- time Stamp
- Message/comment
- like
- etc

Reply:
If you just mean the order of appearance don't worry me and Aisha are looking at the css this afternoon and this will be included in that.